packages/next/package.json

```diff
@@ -94,10 +98,13 @@
   },
   "peerDependencies": {
     "@opentelemetry/api": "^1.1.0",
+    "@playwright/test": "^1.35.1",
```

Comment:
We can omit these peerDependencies for now and add them when stabilizing if desired so users aren't suggested to install.

Reply:
We can move to a `devDependency` so we can pass type checking/build